src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -33,20 +39,20 @@ public EditCommand<?> parse(String args) throws ParseException {
                 String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditCommand.MESSAGE_USAGE));
         }
 
-
         String entityType = trimmedArgs.substring(0, 1);
         String arguments = trimmedArgs.substring(1);
 
         String[] splitArgs = arguments.split("\\s+");
 
-
-        if (splitArgs.length < 2) {
-            throw new ParseException(EditCommand.MESSAGE_NOT_EDITED);
+        if (splitArgs.length < 1) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditCommand.MESSAGE_USAGE));
```

Comment:
Consider adding tests for exceptions, i feel like that's the more important ones as we don't want our app to crash